content/en/platform/corda/5.0/application-networks/creating/optional/mutual-tls-connections.md

```diff
@@ -14,7 +14,7 @@ section_menu: corda5
 
 Corda 5 uses TLS to secure a connection between two clusters. While establishing a TLS connection between the gateways of two clusters, the server gateway sends its certificate to the client gateway. The client gateway verifies the server certificate using its trust root certificate. In mutual TLS, in addition to the client verifying the server certificate, the server gateway also requests the client gateway send a client certificate and verifies that it is using its trust root certificate.
 
-As the gateway manages the TLS connections for an entire cluster, the TLS mode (mutual or one-way) is defined in the gateway configuration and applies to the entire cluster. As a result, any group hosted in a mutual TLS cluster must be a mutual TLS group, and all its members must be hosted on a mutual TLS cluster.
+As the gateway manages the TLS connections for an entire {{< tooltip >}}cluster{{< /tooltip >}}, the TLS mode (mutual or one-way) is defined in the gateway configuration and applies to the entire cluster. As a result, any group hosted in a mutual TLS cluster must be a mutual TLS group, and all its members must be hosted on a mutual TLS cluster.
```

Comment:
how about tooltip for TLS

Reply:
As per the PR comment, this will come in another PR.